.github/workflows/wpt-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: WebPageTest Test Cases
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: npm install webpagetest && npm install jest
+
+      - name: Run WebPageTest
+        run: |
+          EXPECTED_TESTS=$(for file in $(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} \
+            ${{ github.event.pull_request.head.sha }} | grep -E "^dist/.*\.js$"); do basename "$file"; done | cut -d\. -f1 | sort | uniq)
+          CUSTOM_METRIC_FILES=("00_reset", "a11y" "almanac" "aurora" "cms" "css-variables" "css" \
+            "ecommerce" "example" "fugu-apis" "javascript" "markup" "media" "observers" "origin-trials" \
+            "parsed_css" "performance" "privacy" "pwa" "responsive_images" "robots_meta" "robots_txt" \
+            "sass" "security" "structured-data" "third-parties" "valid-head" "well-known" "wpt_bodies")
+
+          # These tests are too big to run using WPT API (custom metrics code needs to be less than 8Kb)
```

Comment:
GET request URL accepted by WPT server is around 6Kb long.
We can start with limited support and see how helpful the tests are.